docs/setup-and-installing.md

```diff
@@ -1,137 +1,36 @@
-
-- This installs the [SpatialOS CLI]({{urlRoot}}/content/glossary#spatial-command-line-tool-cli) , the [SpatialOS Launcher]({{urlRoot}}/content/glossary#launcher), and 32-bit and 64-bit Visual C++ Redistributables
-
-**Step 4.** Install a **code editor** if you don't have one already
-
-- We recommend either [Visual Studio](https://www.visualstudio.com/downloads/) or [Rider](https://www.jetbrains.com/rider/).
-
-**Using Visual Studio?**
-
-Once you have installed [Visual Studio](https://www.visualstudio.com/downloads/), within the Visual Studio Installer, select **.NET Core + ASP .NET Core**.
-
-**Using Rider?**
+To get set you'll need to create a SpatialOS account, install some dependencies and clone the GDK repository.
 
-Once you have installed [Rider](https://www.jetbrains.com/rider/), install the [**Unity Support** plugin](https://github.com/JetBrains/resharper-unity) for a better experience.
+<%(Include file="content/initial-set-up.md")%>
 
-<%(/Expandable)%>
+### Clone the SpatialOS GDK for Unity repository
 
-<%(Callout type="alert" message="
-* You need **Linux** build support. This is because server-workers in a cloud deployment run in a Linux environment. In the `Assets/Fps/Config/BuildConfiguration`, do not change the `UnityGameLogic Cloud Environment` from Linux.<br/>
-* You need **Mac** build support if you are developing on a Windows PC and want to share your game with Mac users.<br/>
-* You need **Windows** build support if you are developing on a Mac and want to share your game with Windows PC users. <br/>
-* Unity gives you build support for your development machine (Windows or Mac) by default.")%>
+To run the GDK for Unity, you need to download the source code. To do this, you need to clone the GDK repository.
 
-<%(#Expandable title="Having problems?")%>
-If you need help using the GDK, please come and talk to us about the software and the documentation via:
+#### Using a terminal:
 
-**The SpatialOS forums**
+Clone the SpacialOS GDK for Unity repository using one of the following terminal commands:
 
-Visit the [support section](https://forums.improbable.io/new-topic?category=Support&tags=unity-gdk) in our forums and use the unity-gdk tag.
+|     |     |
+| --- | --- |
+| SSH | `git clone git@github.com:spatialos/gdk-for-unity.git` |
+| HTTPS | `git clone https://github.com/spatialos/gdk-for-unity.git` |
 
-**Discord**
+#### Using GitHub Desktop
 
-Find us in the [#unity channel](https://discord.gg/SCZTCYm). You may need to grab Discord [here](https://discordapp.com/).
+To clone the repository using GitHub desktop: navigate to **File > Clone Repository**, click the URL tab and enter `https://github.com/spatialos/gdk-for-unity` into the URL or username/repository field, then click Clone. 
 
-**Github issues**
+#### Checking out the latest release
```

Comment:
```suggestion
#### Check out the latest release
```
This one is an instruction if you are using GitHub, I think, so we lose the gerund ("ing").